### ISSUE TITLE
the struct palette_rgb_5_6_5 in types.h 

### ISSUE THREAD

**Issue by icecity96** [uascorpion/jpeg_algorithm#1]
In struct `palette_rgb_5_6_5` why has a number `unused`?I think the right struct should be
```
typedef struct {
    byte_t blue   : 5;
    byte_t green  : 6;
    byte_t red    : 5;
} Palette_rgb_5_6_5;
```
see [here](https://www.theimagingsource.com/support/documentation/ic-imaging-control-cpp/PixelformatRGB565.htm) about rgb565 memory layout

**Comment by uascorpion** [comment]
Yes, you are right. It was a copy-paste mistake...

**Comment by uascorpion** [comment]
Fixed in commit 9ec670c
